docs/source/index.rst

```diff
@@ -32,49 +31,54 @@ You can easily deploy Sycamore locally or on a virtual machine using Docker.
 
 With Docker installed:
 
-
-
-1. Clone the Sycamore repo:
+1.	Clone the Sycamore repo: 
 
 ``git clone https://github.com/aryn-ai/sycamore``
 
-2. Set OpenAI Key:
+2.	Set OpenAI Key:
 
 ``export OPENAI_API_KEY=YOUR-KEY``
 
-3. Go to:
+3.	Go to: 
```

Comment:
Can you add a task to the sprint to figure out why linting isn't finding the trailing whitespace?  It's not complaining but when I run the bigger set of precommit checks it's going to (and reformat it).

Reply:
Let's bring this up in stand-up tmrw.